presto-product-tests/conf/presto/etc/singlenode-ldap.properties

```diff
@@ -17,13 +17,14 @@ discovery-server.enabled=true
 discovery.uri=http://presto-master:8080
 
 # LDAP specific properties
-# https will have to enabled for ldap authentication
+# https will have to be enabled for ldap authentication
 http-server.https.port=8443
 http-server.https.enabled=true
 http-server.https.keystore.path=/etc/ldap/coordinator.jks
 http-server.https.keystore.key=testldap
 authentication.ldap.enabled=true
-authentication.ldap.url=ldaps://ldap-server:636
-authentication.ldap.ad-domain=presto.testldap.com
-authentication.ldap.base-dn=OU=Asia,DC=presto,DC=testldap,DC=com
-authentication.ldap.group-dn=CN=ParentGroup,OU=America,DC=presto,DC=testldap,DC=com
+authentication.ldap.server-type=OPENLDAP
+authentication.ldap.user-object-class=inetOrgPerson
+authentication.ldap.url=ldaps://ldapserver:636
+authentication.ldap.base-dn=ou=Asia,dc=presto,dc=testldap,dc=com
+authentication.ldap.group-dn=cn=DefaultGroup,ou=America,dc=presto,dc=testldap,dc=com
```

Comment:
I see you have created the DefaultGroup in openLDAP. Did you also create it in AD?